ccc-js/src/main/javascript/package-res/ccc/core/base/sign/base-sign.js

```diff
@@ -159,9 +159,14 @@ def('pvc.visual.BasicSign', def.Object.extend({
             var state;
             if(createIndep || !(state = this.instanceState())) return this._createContext(scene);
 
-            return state.cccContext || (state.cccContext = this._createContext(scene));
+            var cccContext;
+            if((cccContext = state.cccContext) == null) cccContext = state.cccContext = this._createContext(scene);
+
+            if(cccContext.event == null && pv.event != null) cccContext.event = pv.event;
```

Comment:
Have you checked if pv.event is defined at this point: https://github.com/davidmsantos90/ccc/blob/master/ccc-js/src/main/javascript/package-res/ccc/core/base/sign/sign.js#L541 ?

Reply:
in this case **wait** is false, so that branch of code is not executed, instead it goes here https://github.com/davidmsantos90/ccc/blob/master/ccc-js/src/main/javascript/package-res/ccc/core/base/sign/sign.js#L527